src/engine/enginemaster.cpp

```diff
@@ -155,14 +171,17 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
     m_pKeylockEngine->set(pConfig->getValueString(
             ConfigKey(group, "keylock_engine")).toDouble());
 
+    // TODO: Make this read only but let EngineMasterTest enable it
```

Comment:
Yes, that would be nice. I thought about it when I have introduced this preference option. But it was too hard to find clear indicators if we need the master mix or not. This option was a workaround. 
I think we should keep this workaround working in this PR for now.